lessons/Data-Science/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,278 @@
+
+@centered-image{images/iris-data.png}
+
+=== Synthesize
+
+@QandA{
+
+@Q{How is playing 20 questions to figure out a mystery item _similar_ to the predicting and recommending that a decision tree enables?}
+@A{Decision trees, like 20 Questions participants, need to avoid questions that don't provide information. Decision trees, like 20 Questions participants, sometimes fail to identify an object correctly. Familiarity with the dataset improves the accuracy of both a decision tree and a 20 Questions participant.}
+
+@Q{How is it different?}
+@A{When we play 20 Questions, we are trying to identify a single object. Decision trees are useful for categorizing a variety of objects, or predicting outcomes in a variety of different scenarios.}
+}
+
+
+
+== Decision Trees from Training Datasets @duration{25 minutes}
+
+=== Overview
+
+Students build a decision tree that predicts whether different individuals will purchase a video game or not.
+
+=== Launch
+
+We have already built some extremely simple decision trees. We have a sense of the hierarchical structure, flexibility, and versatility of decision trees. We know that decision trees sometimes fail when provided with ambiguous or unknown data. We understand the risk of overfitting our data (building a model that can only succeed on narrow set of data).
+
+We have *not* yet learned about how decision trees built from large datasets make relatively accurate predictions, recommendations, and diagnoses.
+
+The key to building an effective decision tree is to decide—at every level and every node—which attributes are the most informative ones to ask questions about.
+
+...but how do we make such decisions?!
+
+It turns out, there's an algorithm for that, and it's relatively straightforward.
+
+
+=== Investigate
+
+Have you ever done some online shopping—say, for a new pair of sneakers—only to discover that, for the next several days, you encounter _advertisements for sneakers_ lurking in every corner of the internet that you visit?!
+
+Is it a coincidence? No. Computer cookies are small data files stored locally on your device. One particular kind of cookie, the tracking cookie, allows AI designed for marketing to use your individual browsing habits to decide which ads you will be the most susceptible to.
+
+We're going to create a decision tree that predicts whether or not different customers at a particular online store will purchase a video game or not. To do so, we must first train the computer! We will use a training dataset that characterizes 14 different shoppers and then indicates whether each one purchased a video game or not.
+
+@QandA{
+@Q{With your partner, look over the @handout{decision-tree-data.adoc, Training Dataset}. What do you Notice? What do you wonder?}
```

Comment:
I think it might be useful to include 3-4 rows from this table in the lesson plan, and add text that explains what each column is about. Remember that this will also result in the table showing up on the lesson slide, which I think will be valuable.

Reply:
originally I had the whole table and it was huge. If I include 3-4 rows, is there anyway we can reduce their height? Agree that we definitely want the table in the slides.